coyote/views.py

```diff
@@ -24,6 +24,11 @@ def get_domain(url):
 
 
 def home(request):
+    """
+    it takes necessary data from frontend request and sent it to hackerearth API
+    if the code doesnt compiles, then it takes necessary keyword from error messages 
+    and google the result with some regrex matching and filtering to get required debug links 
```

Comment:
* and searches for it on google with regex (spelled regrex) matching 
* remove _some_
* required => helpful or _get required_ => suggest

Reply:
The documentation cannot contain words like some, required. Documentation must be deterministic, leaving nothing to chance.